python/ray/air/util/check_ingest.py

```diff
@@ -71,7 +71,7 @@ def preprocess_datasets(self):
                 start = time.perf_counter()
                 # Force execution to time preprocessing since Datasets are lazy by
                 # default.
-                ds.cache()
+                ds.materialize()
```

Comment:
The original code path here looks like to force excution of all `ds` in `self.datasets`. Should we use `ds.fully_executed()` here (and fix it in 2.5)? I am not sure if this will cause any regression that `self.datasets` will re-execute later.

Reply:
Fixed this. It does seem like it would change timings otherwise.